screens/DoubleSearch.js

```diff
@@ -62,7 +62,19 @@ function DoubleSearch(props) {
      * Description: Method to go back to the previous screen.
      * Using Stack navigator.
      */
+
+    // var fromScreen; 
+    CourseScheduleDetailsScreen = props.navigation.getParam("CourseScheduleDetailsScreen", "null");
+    NearbyInterestDetailsScreen = props.navigation.getParam("NearbyInterestDetailsScreen", "null");
     const goBack = () => {
+        if(CourseScheduleDetailsScreen === true){
+            props.navigation.goBack();
+            props.navigation.navigate("CourseScheduleDetails")
+        }
+        else if (NearbyInterestDetailsScreen === true){
+            props.navigation.goBack();
+            props.navigation.navigate("NearbyInterestDetails")
+        }
         props.navigation.goBack();
```

Comment:
If both are false, line 78 `props.navigation.goBack();` goes back where?

Reply:
If both are false, then it goes back to either mapView or MoreDetails (it goes back to the screens it originated from, thats within the same navigation stack)

Reply:
Wouldn't it be better to add this in an _else_ bracket in that case?

Reply:
Will do!